src/components/DonationDialog.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <div>
+    <q-dialog v-model="inDonation">
+      <q-card class="donation-card">
+        <q-card-section>
+          <h6 class="q-my-sm">Thank you for donating! 💕</h6>
+          <span>Your donations help keep our work alive.</span>
+          <span><br>All proceeds go to Grey Software.</span>
+        </q-card-section>
+        <q-card-section>
+          <div class="column items-center">
+            <q-btn
+              rounded
+              class="q-pa-xs q-ma-sm"
+              icon="img:../statics/icons/paypal-28x28.png"
```

Comment:
Since merging master moved the icons to /public/icons, we can now remove /statics/

So the path should be /icons/paypal-28x28.png

Reply:
done